product_manufacturer/tests/test_product_manufacturer.py

```diff
@@ -8,7 +8,6 @@ class TestProductManufacturer(TransactionCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Comment:
Either you let this, either you inherit from BaseCommon https://github.com/odoo/odoo/blob/17.0/odoo/addons/base/tests/common.py#L21

This allows to enhance tests performances.